contribution/general.md

```diff
@@ -0,0 +1,251 @@
+---
+layout: sideMenu
+title: "General"
+description: ""
+menu: nav-contrib
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Contribution Guidelines
+
+**Zeppelin** is [Apache2 License](https://github.com/apache/zeppelin/blob/master/CONTRIBUTING.md) Software.
```

Comment:
I'm pretty sure a title/subtitle/first word of the page needs to be "Apache project" http://www.apache.org/foundation/marks/pmcs.html#naming